### ISSUE TITLE
Warning: Expected `onClick` listener to be a function, instead got a value of `string` type.

### ISSUE THREAD

**Issue by Gerwyn1** [adrianhajdin/ecommerce_sanity_stripe#5]
Unable to add different items to cart, always add same/initial item. i have cloned this project and still not work.

![image](https://user-images.githubusercontent.com/52874400/165380527-b68d3dae-b934-43c2-918e-6e0b79d48f06.png)

**Comment by taiseen** [comment]
in **Cart** component at line number 78, remove that **onClick=""**, then you don't get this error...

**Comment by Gerwyn1** [comment]
the error is gone but sadly im still unable to add different cart items. any idea?

**Comment by taiseen** [comment]
I'm not sure why it's happening at your end... so let's add some screen short here to see what is going on at your project?

**Comment by Gerwyn1** [comment]
Again i've added the 2nd item (x2) and 3rd item (x3) and it added the first item (x5) to the cart. also the console has no more errors.

![image](https://user-images.githubusercontent.com/52874400/165388221-692f39ac-fc5c-4398-8bf4-3fc2f0375814.png)

**Comment by taiseen** [comment]
Don't mind... I think you may have missed conception to understand this thing because that 2nd or 3rd item you **can't add** because they are already part of the 1st product item. These pictures are just another-looking version of that specific product that you already added to your cart & after adding the product into the cart you can only see just 1st picture from that product. 

So it's not an error, it's a feature behavioral pattern 🙂

**Comment by Gerwyn1** [comment]
But how? Just before I copy pasted the github code, it was working fine.

'These pictures are just another-looking version of that specific product that you already added to your cart'

But I manually added the items to sanity CMS.

I'm guessing the github source code actually has bugs

**Comment by beaxion** [comment]
From what I see you need to create a separate products in Sanity dashboard, not just add different pictures to the same product.

**Comment by Gerwyn1** [comment]
> From what I see you need to create a separate products in Sanity dashboard, not just add different pictures to the same product.

Nope. Didn't work. I've added different products once again (this time smartwatch) 

![image](https://user-images.githubusercontent.com/52874400/165503717-4c9c012c-dc1a-4d4b-9cef-074079be1b8e.png)

![image](https://user-images.githubusercontent.com/52874400/165503802-cad13fec-4e87-412a-8f6d-75428432b8b5.png)

Also, i followed the exact steps in the tutorial video, what adrian did. 

and by the way i been having this bug too which im unable to solve (ain't easy man):
![image](https://user-images.githubusercontent.com/52874400/165504153-f97123de-0d26-4877-9386-7610f97bdbed.png)

**Comment by meetsmarttechnica** [comment]
same problem its a bug when add one item than add another item multiple time its is item._id is is undefianed why

**Comment by Pannu786** [comment]
Hello soldiers we could use some extra pairs of eyes for the issue I am facing. Please help me with resolving my errors. I have already posted the issue above. Thank you.
